### PR TITLE
add OWNERS for logging

### PR DESCRIPTION
**Pull request by richm**
add OWNERS for logging

### CODE REVIEW

**Comment by richm** [comment]
@jcantrill ptal - not sure who is already on this list

**Comment by stevekuznetsov** [comment]
Sweet, thanks
/lgtm
/hold

@jcantrill feel free to `/hold cancel` when you want this to merge

File: ci-operator/jobs/openshift/origin-aggregated-logging/OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.mdapprovers:
+  - jwforres
+  - jcantrill
+  - ewolinetz
+  - richm
+  - lukas-vlcek
+  - nhosoi
+  - josefkarasek
+  - nkinder
```
[the remark this answers is not in the document]
Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/origin-aggregated-logging/OWNERS]: @matthyx amazing :)

**Pull request merged**